core/templates/pages/exploration-editor-page/editor-tab/services/solution-verification.service.spec.ts

```diff
@@ -56,8 +57,21 @@ describe('Solution Verification Service', function() {
         display_mode: 'inline',
         is_terminal: true
       }
+    };
+
+    TestBed.configureTestingModule({
+      providers: [
+        { provide: INTERACTION_SPECS, useValue: mockInteractionState }
+      ]
     });
-  }));
+
+    siis = TestBed.get(StateInteractionIdService);
```

Comment:
@subedi90 can you create an issue saying to change `ees = $injector.get(...)` to `TestBed.get` when ExplorationStateService has been migrated? After doing that could add TODO(#{{issue number}}): Change $injector to TestBEd.get?

Reply:
Can we do it when we remove usaeg of UpgradeServices from this file ? There is a TODO for removing UpgradeServices on top of this file, and I believe that would be done at the same time when we are ready to remove $injector.get(...) ?